app/plugins/hmac-authentication.md

```diff
@@ -67,30 +82,33 @@ A [Consumer][consumer-object] can have many credentials.
 
 ### Create a Credential
 
-You can provision new username/password credentials by making the following HTTP request:
+You can provision new username/password credentials by making the following
+HTTP request:
 
 ```bash
 $ curl -X POST http://kong:8001/consumers/{consumer}/hmac-auth \
     --data "username=bob" \
     --data "secret=secret456"
 ```
 
-`consumer`: The `id` or `username` property of the [Consumer][consumer-object] entity to associate the credentials to.
+`consumer`: The `id` or `username` property of the [Consumer][consumer-object]
+entity to associate the credentials to.
 
 form parameter             | description
 ---                        | ---
 `username`                 | The username to use in the HMAC Signature verification.
-`secret`<br>*optional*   | The secret to use in the HMAC Signature verification. Note that if this parameter isn't provided, Kong will generate a value for you and send it as part of the response body.
+`secret`<br>*optional*     | The secret to use in the HMAC Signature verification. Note that if this parameter isn't provided, Kong will generate a value for you and send it as part of the response body.
 
 ### Signature Authentication Scheme
 
-The client is expected to send an `Authorization` header with the following parameterization:
+The client is expected to send an `Authorization` or `X-Authorization` header
```

Comment:
According to the documentation (and the code), should it not be 'Authorization' or 'Proxy-Authorization'?